### ISSUE TITLE
Narrow widget layout

### ISSUE THREAD

**Issue by rstruminski** [dmfs/opentasks#175]
When set to a rather high and narrow layout, the widget only displays the tasks' dates, but not the titles (see screenshot).

I think it would make more sense if it displayed the titles in this case, and ignored the dates.

![screenshot_2015-11-07-21-43-51](https://cloud.githubusercontent.com/assets/1116399/11017276/11d7b50a-859a-11e5-8942-23ea0851872b.png)

**Comment by dmfs** [comment]
Good point. It's time for a major overhaul of the widget. We already have concrete plans and a new widget design. We'll check if it plays well with the narrow layout and adjust if necessary.

**Comment by filipesaraiva** [comment]
Thanks @dmfs, a new widget design will be amazing for the app. Currently, only OpenTasks can to sync and present tasks from CalDAV calendars. If possible, give a look at [Chronus Task Widget](https://play.google.com/store/apps/details?id=com.dvtonder.chronus) (only Google Task support) to inspire the new OpenTasks widget design.

![Chronus Task Widget](https://lh3.googleusercontent.com/-fGVQaKW1cCM/Vcnfq_Yi0uI/AAAAAAAAIc8/m4jIf50X0to/s319-p/Screenshot_2015-08-06-07-09-44.png)

:)
